cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -94,6 +94,15 @@ spec:
         # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
         # - name: AGENT_TOLERATION_KEY
         #   value: "<KeyOfTheTaintToTolerate>"
+        # (Optional) Rook Agent NodeAffinity Label Key. 
+        # Set this to the key of the Node Label to match.
+        # - name: AGENT_NODE_LABEL_KEY
```

Comment:
An example might help here. For example, the [cluster CR example](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/cluster.yaml#L56-L63) uses `role=storage-node` as the node affinity sample.

Reply:
ack. will update.